src/main/java/me/gallowsdove/foxymachines/implementation/materials/ShardMaterial.java

```diff
@@ -30,6 +30,7 @@ public ItemDropHandler getItemHandler() {
             }
 
             if (!QuestUtils.hasActiveQuest(p)) {
+                p.sendMessage(this.color + "You have no active quest!");
```

Comment:
I think a better message would be something like "you should check your quest with /foxy quest first!". After that, there should be no point at which one doesn't have a quest I believe